graphz/src/main/scala/coop/rchain/graphz/Graphz.scala

```diff
@@ -1,104 +1,104 @@
 package coop.rchain.graphz
 
 import java.io.FileOutputStream
-
 import cats._
-import cats.effect.Sync
+import cats.effect.{Concurrent, Sync}
+import cats.effect.concurrent.Ref
 import cats.syntax.all._
-import cats.mtl._
 
 trait GraphSerializer[F[_]] {
   def push(str: String, suffix: String = "\n"): F[Unit]
+  def show: F[String]
```

Comment:
The point of this trait is to abstract over logic how final result is constructed by calling `push` method.
This means that we don't want to have methods like `show` because whoever is creating the instance will have full control to collect all and get the final result.

This definition of `show` also implies that we must have String as a result which is doesn't have to be the case when writing to as file or list of string lines.

Reply:
Fixed in [commit](https://github.com/rchain/rchain/pull/3574/commits/fac58f356148e6c47d8188344c1b6364a931c3f5).